tools/cloud-workstations/Dockerfile

```diff
@@ -50,14 +50,19 @@ RUN pip install --no-cache-dir --upgrade pip && \
 
 # Get the HPC config files and store them in the correct locations for startup
 ARG CW_DIR=tools/cloud-workstations
-ARG HPC_FILE=configure-hpc-toolkit.sh
-ARG HPC_CONF_FILE=030_configure-hpc-toolkit.sh 
+ARG HPC_WORK_FILE=200_configure-hpc-toolkit.sh
+ARG HPC_CONF_FILE=configure-hpc-toolkit.sh
+ARG CODE_REQS=code_oss_requirements.txt
+ARG HPC_CONFIG_DIR=/etc/hpc-toolkit-config
+ARG START_DIR=/etc/workstation-startup.d
+ARG INSTALL_DIR=/usr/local
 
-COPY $CW_DIR/$HPC_FILE /bin/
-COPY $CW_DIR/$HPC_CONF_FILE /etc/workstation-startup.d/
+COPY $CW_DIR/$HPC_CONF_FILE $INSTALL_DIR/bin
+RUN chmod a+rx $INSTALL_DIR/bin/$HPC_CONF_FILE
```

Comment:
We could avoid `RUN chmod` by either:
* setting proper permission at the source (in the repo), since `COPY` preserves permissions;
* use `COPY --chmod=...`.

Reply:
Tested this out.  It's not native to the version of Docker that's running, so you need to set an environment variable `DOCKER_BUILDKIT=1` on the yaml file.  It seems to slow down the build, so I don't think we should change this.